djangocms_navigation/utils.py

```diff
@@ -0,0 +1,26 @@
+import functools
+
+from functools import lru_cache
+
+from django.apps import apps
+
+
+@lru_cache(maxsize=1)
+def supported_models():
```

Comment:
Wouldn't it make more sense if this returned a queryset of content type objects? I mean otherwise this is just an alias for `extension.navigation_apps_models`, no?